src/main/java/lesson02/tasksresolve/PrimeChecker.java

```diff
@@ -0,0 +1,56 @@
+package lesson02.tasksresolve;
+
+import lesson02.utility.ConsoleInput;
+
+/*
+Task 5 (PrimeChecker)
+Определить, является ли число простым. Простое число - это натуральное число (больше нуля), которое делится нацело
+только на себя и на 1.
+Ограничения: n > 0.
+boolean isPrime(int n)
+Входные аргументы: int n
+Тип результата: boolean
+ */
+public class PrimeChecker {
+    private static final String MESSAGE_FOR_NUMBER_INPUT = "Please input integer number that greater than 0: ";
+    private static final String MESSAGE_FOR_IS_A_PRIME = "Input number is prime";
+    private static final String MESSAGE_FOR_IS_NOT_A_PRIME = "Input number is not a prime";
+    private static final String MESSAGE_FOR_NUMBER_IS_LESS_THAN_ZERO = "Number should be > 0";
+
+    public static void main (String[] args) {
+        if (isPrime(ConsoleInput.inputNumberFromConsole(MESSAGE_FOR_NUMBER_INPUT))){
```

Comment:
I suggest moving to the local variable result of the input from console.

Reply:
Fixed